src/App.jsx

```diff
@@ -84,14 +84,31 @@ const PreviewData = [
         'title': 'The Central Limit Theorem',
         'text': 'In probability theory, the central limit theorem (CLT) establishes that, in some situations, when independent random variables are added, their properly normalized sum tends toward a normal distribution (informally a "bell curve") even if the original variables themselves are not normally distributed.'
     },
+    {
+        'slug': '/linear-regression-model',
+        'title': 'Linear Regression Model',
+        'text': 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce pharetra lorem blandit, blandit lectus vel, tincidunt leo. Donec sagittis eleifend purus at egestas. Praesent eu mattis est. Etiam et lectus sed libero cursus cursus. Class aptent taciti socios quad litora torquent per conubia nostra, per inceptos himenaeos. Integer a dignissim justo. Suspendisse gravida lectus eget tincidunt.',
+    },
```

Comment:
Hey @mrcrymnd I refactored a few things.  The `<Preview />` below maps over this array and renders the markup.  To add more cards or update the text, change it here.